repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEnumDefStoreV2.java

```diff
@@ -54,6 +54,11 @@ public AtlasVertex preCreate(AtlasEnumDef enumDef) throws AtlasBaseException {
 
         validateType(enumDef);
 
+        AtlasTypeRegistry invalidKeywordsForTypeCreation = new AtlasTypeRegistry();
```

Comment:
No need to declare whole typeRegistry again, watch out for already existing object of TypeRegistry and use that, would be accessible via this.typeRegistry